getting-started/installation-timescale-cloud.md

```diff
@@ -0,0 +1,16 @@
+## Installation (Timescale Cloud) [](installation-timescale-cloud)
+
+Timescale Cloud is a Database as a Service (DBaaS) offering that provides an easy way for you to store and analyze time-series. Powered by TimescaleDB, you can create database instances in the cloud and automate many of your most common operational tasks. 
```

Comment:
solar hates those long lines, could you add linebreaks here for less than 80 char lines

Reply:
Done!